payment-failure/src/main/scala/com/gu/identity/paymentfailure/BrazeClient.scala

```diff
@@ -2,31 +2,32 @@ package com.gu.identity.paymentfailure
 
 import com.typesafe.scalalogging.StrictLogging
 import cats.syntax.either._
+import io.circe.parser.decode
 import io.circe.syntax._
 import scalaj.http.Http
 
 
 class BrazeClient(config: Config) extends StrictLogging {
 
-  def sendEmail(emailData: IdentityBrazeEmailData, emailToken: String) : Either[Throwable, BrazeResponse] = {
+  def sendEmail(request: BrazeSendRequest) : Either[Throwable, BrazeResponse] = {
```

Comment:
Have the email service be responsible for constructing a Braze request; the client is just concerned about executing it.